### DISCUSSION TITLE
Welcome to base-custom-webcomponent Discussions!

### DISCUSSION THREAD

**Discussion by jogibear9988** [node-projects/base-custom-webcomponent#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KhalifRich** [comment]
This is Khalif,undergraduate majoring in Software Eng,Kenyan citizen.I would require review assistance of PHP code.
